corehq/form_processor/management/commands/run_sql.py

```diff
@@ -30,20 +32,24 @@ def add_arguments(self, parser):
         parser.add_argument('-d', '--dbname', help='Django DB alias to run on')
         parser.add_argument('--chunk-size', type=int, default=1000,
             help="Maximum number of records to move at once.")
+        parser.add_argument('--ignore-rows',
+            action="store_false", dest="print_rows", default=True,
```

Comment:
I liked the way the logic works out with `print_rows` variable (otherwise we'd have `if not ignore_rows:` later in the file. Also, I had implemented `--print-rows` first, but then realized that I want it to be enabled by default.